plugins/parsers/registry.go

```diff
@@ -53,6 +54,9 @@ type Config struct {
 
 	// DefaultTags are the default tags that will be added to all parsed metrics.
 	DefaultTags map[string]string
+
+	// PromFormat only to Prometheus
+	PromFormat map[string]string
```

Comment:
Ok ! You're right, it's http header...
How can I pass this stuff to the Prometheus parser ?